Converted to using scalatest.

tfenne opened the pull request:
@cmnbroad here's a cut at converting your URI-spec test over to scalatest and getting gradle to run them.

tfenne committed:
add code of conduct (#44)

* Create CODE_OF_CONDUCT.md

magicDGS committed:
Update .gitignore for Eclipse (#40)

magicDGS committed:
Add badges to README and AppVeyor configuration (#24)

- Add build badges (Travis-CI and AppVeyor)
- Add LICENSE and language badges
- Add configuration for AppVeyor to test in Windows systems

tfenne committed:
Converted to using scalatest.

cmnbroad commented:
Great thx for doing that. I haven't used scala in several years so I'll have to spend some time getting back up to speed, and then will be back with questions.

cmnbroad committed:
URI and Path specifier classes for input/output resources. (#34)

Merging an initial but contested proposal in order to have something  concrete we can work on.

* URI and Path specifier classes for input/output resources.

* Changes for Windows and test refactoring.

tfenne committed:
Restoring test data files that were lost.

tfenne committed:
Removed trailing newlines from test data files.
